docs/DevGuide/QuickStart.md

```diff
@@ -0,0 +1,244 @@
+\cond NEVER
+Distributed under the MIT License.
+See LICENSE.txt for details.
+\endcond
+# Code development quick-start {#dev_guide_quick_start}
```

Comment:
I don't think this should be the "quickstart" guide. We should not be imposing a specific IDE or development workflow...

Reply:
How about @quick start with docker & visual studio code" along with language that this is a quick start, not the quick start?

Reply:
That's fine by me